### ISSUE TITLE
How to Set the Threshold for gte-multilingual-reranker

### ISSUE THREAD

**Issue by ketusrai** [huggingface/text-embeddings-inference#461]
I want to use the gte-multilingual-reranker-base model to re-rank the retrieved documents and discard some of them based on a threshold. I have seen examples on Hugging Face where the logits are used as the output scores, but how can I determine the appropriate threshold?

**Comment by HiddenPeak** [comment]
in your application？

**Comment by ketusrai** [comment]
Yes. In my application.
